test_data/yast/yast2_gui.yaml

```diff
@@ -0,0 +1,11 @@
+bootloader:
+  bootcode_options:
+    bootcode_options_tab_selected: '1'
```

Comment:
this is something that depends on the navigation and the first tab that is selected when the module is open. This doesn't look the right place to put it, in the test data, which should not care about 'tabs' (UI controls) neither.
Widely in our code we are adding method to wait for some specific control to ensure that the screen is loaded, so for example you can add in `get_bootcode_options_page` something like in "/v4_3/GuidedSetupController.pm":
```
die "Partitioning Scheme is not displayed" unless $self->{PartitioningSchemePage}->is_shown();
```

Reply:
The logic of this is to verify what things are pre-selected by default, in this case, what tab is pre-selected.

Reply:
the logic is guarantee by checking some control in that method, you are thinking ahead about how to check what tab is selected, but for the problem you have, you don't need that. Every tab would be a page, and every page would have is_shown pointing to some unique controls, but that is out of scope, and definitively doesn't belong to test data, we need to code that behavior, that when we enter we interact with that page, if the behavior change in other product we can have some function in the controller that we override.